lib/devices/device_descriptors/schwinn_x70.dart

```diff
@@ -94,6 +116,14 @@ class SchwinnX70 extends FixedLayoutDeviceDescriptor with CadenceMixin, PowerSpe
     clearCadenceData();
   }
 
+  // Adapted from 
+  // https://github.com/ursoft/ANT_Libraries/blob/e122c007f5e1935a9b11c05e601a71f2992bad45/ANT_DLL/WROOM_esp32/WROOM_esp32.ino#L525
+  double powerFromCadenceResistance(int cadence, int resistance) {
+    final int idx = (resistance - 1) % 25;
+    final double ret = resistancePowerCoeffs[idx][0] * cadence * cadence + resistancePowerCoeffs[idx][1] * cadence + resistancePowerCoeffs[idx][2];
+    return ret > 0.0 ? ret : 0.0;
```

Comment:
In other places I operated with the `min()` and `max()` function to limit values. I think we should be consistent across the code and possibly use `max(power, 0)` to avoid negative values, however I understand that the ternary operator might be more descriptive to some. Let me know what you think.

Reply:
You're right.  A consistent style makes the code easier to understand.  Fixed.

Reply:
I'm looking again and I really want to merge all your PRs. I'll possibly devise PRs to your branches with the modifications to help merge the changes into my code. I really appreciate your efforts

Reply:
I crafted a PR for all 3 of your PRs, we can use those to make your PRs merge-able